### PR TITLE
Added algorithm for finding cycle in undirected graph

### PR DESCRIPTION
**Pull request by anand4234**

### CODE REVIEW

File: Graph Algorithm/detect_cycle_ug.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#define NODE 5
+using namespace std;
+
+int graph[NODE][NODE] = {
+   {0, 1, 0, 0, 0},
+   {1, 0, 1, 1, 0},
+   {0, 1, 0, 0, 1},
+   {0, 1, 0, 0, 1},
+   {0, 0, 1, 1, 0}
+};
```
Comment (siddharth25pandey) [inline Graph Algorithm/detect_cycle_ug.cpp:13]: Take Dynamic Input from the user

File: Graph Algorithm/detect_cycle_ug.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#cpp program to detect cycle in an undirected graph.
+#include<iostream>
+#include<set>
```
Comment (siddharth25pandey) [inline Graph Algorithm/detect_cycle_ug.cpp:3]: Use single header file **bits/stc++**

File: Graph Algorithm/detect_cycle_ug.cpp [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#cpp program to detect cycle in an undirected graph.
+#include<iostream>
+#include<set>
+#define NODE 5
```
Comment (siddharth25pandey) [inline Graph Algorithm/detect_cycle_ug.cpp:4]: Take input from the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@anand4234 Do the changes as suggested.
Thanks!

**Comment by anand4234** [comment]
@siddharth25pandey I've changed the code to take graph nodes as input.

**Comment by anand4234** [comment]
Hi @siddharth25pandey I've added Kruskal algorithm also ..pls review

**Pull request closed without merging**